tests/examples/test_download_files.py

```diff
@@ -988,65 +989,76 @@ def test_coil_magnetic_field():
 def test_load_sun():
     mesh = examples.planets.load_sun()
     assert mesh.n_cells
-    assert mesh.textures["atmosphere"]
+    with pytest.warns(PyVistaDeprecationWarning):
+        assert mesh.textures["atmosphere"]
 
 
 def test_load_moon():
     mesh = examples.planets.load_moon()
     assert mesh.n_cells
-    assert mesh.textures["surface"]
+    with pytest.warns(PyVistaDeprecationWarning):
+        assert mesh.textures["surface"]
 
 
 def test_load_mercury():
     mesh = examples.planets.load_mercury()
     assert mesh.n_cells
-    assert mesh.textures["surface"]
+    with pytest.warns(PyVistaDeprecationWarning):
+        assert mesh.textures["surface"]
 
 
 def test_load_venus():
     mesh = examples.planets.load_venus()
     assert mesh.n_cells
-    assert mesh.textures["surface"]
-    assert mesh.textures["atmosphere"]
+    with pytest.warns(PyVistaDeprecationWarning):
+        assert mesh.textures["surface"]
+        assert mesh.textures["atmosphere"]
```

Comment:
Another place where we might want to break this up into two `with` blocks. Maybe not, because the texture lookup logic is the same on both lines.